camus-shopify/script/update_oozie_schedule

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+import os
+import sys
+import json
+import subprocess
+import logging as lg
+from datetime import datetime
+
+from oozie import oozie_server, workflow, coordinators, bundle, actions
+
+formatter = lg.Formatter('%(asctime)s %(levelname)s %(name)s %(message)s', '%d/%m/%y %H:%M:%S')
+ch = lg.StreamHandler(sys.stdout)
+ch.setFormatter(formatter)
+logger = lg.getLogger('oozie_submit_job')
+logger.setLevel(lg.INFO)
+logger.addHandler(ch)
+
+
+def run_shell_command(cmd, description, cwd=None, shell=False):
+    if not cwd:
+        cwd = os.getcwd()
+    p = subprocess.Popen(cmd, cwd=cwd, stdout=subprocess.PIPE, shell=shell)
+    stdout, stderr = p.communicate()
+    logger.info('%s: %s' % (description, stdout))
+    if stderr:
+        logger.error(stderr)
+        exit(-1)
+    else:
+        return stdout.strip()
+
+
+def get_camus_version(path):
+    path = os.path.join(path, 'camus-shopify')
+    logger.info('Retrieving Camus version from %s' % path)
+    cmd = "mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.version | grep -Ev '(^\[|Download\w+:)'"
+    return run_shell_command(cmd, 'Retrieved Camus version', path, True)
+
+
+def main():
+    secrets = json.load(open(os.path.join(os.getcwd(), 'camus-shopify', 'oozie.json')))
+    oozie = oozie_server.OozieServer(secrets['oozie_url'])
+
+    command = 'bash -c "curl --compressed -H \'Accept: application/json\' -X GET \'http://hadoop-rm.chi.shopify.com:8088/ws/v1/cluster/apps?states=RUNNING,ACCEPTED\' | grep -v Camus && echo \'Camus is not running\' && '
+    command += 'java -Xms1G -Xmx2G -DHADOOP_USER_NAME=deploy -Dlog4j.configuration=file:/u/apps/camus/shared/log4j.xml '
+    command += '-cp $(hadoop classpath):/u/apps/camus/current/camus-shopify-0.1.0-shopify1.jar:/etc/camus:/etc/hadoop/conf '
+    command += 'com.linkedin.camus.etl.kafka.CamusJob -P /u/apps/camus/shared/camus.properties"'
```

Comment:
You can remove `/etc/hadoop/conf` from the path as that is the first entry of `hadoop classpath`.